moped-editor/src/views/projects/projectView/ProjectComponents/mapUtils.js

```diff
@@ -104,62 +108,73 @@ export const ProjectComponentsSourcesAndLayers = ({
             ...mapStyles["ctn-lines"].layerProps,
             layout: {
               ...mapStyles["ctn-lines"].layerProps.layout,
-              visibility:
-                isEditingComponent && linkMode === "lines" ? "visible" : "none",
+              visibility: isEditingLines ? "visible" : "none",
             },
           }}
         />
       </Source>
 
       <Source
-        id="ctn-points"
+        id="project-lines"
         type="geojson"
-        data={ctnPointsGeojson}
-        promoteId={SOURCES["ctn-points"]._featureIdProp}
+        data={projectLines}
+        promoteId="id"
       >
         <Layer
           beforeId="street-labels"
           {...{
-            ...mapStyles["ctn-points-underlay"].layerProps,
+            ...mapStyles["project-lines-underlay"].layerProps,
             layout: {
-              visibility:
-                isEditingComponent && linkMode === "points"
-                  ? "visible"
-                  : "none",
+              ...mapStyles["project-lines-underlay"].layerProps.layout,
+              visibility: isEditingLines ? "visible" : "none",
             },
           }}
         />
         <Layer
           beforeId="street-labels"
           {...{
-            ...mapStyles["ctn-points"].layerProps,
+            ...mapStyles["project-lines"].layerProps,
             layout: {
-              visibility:
-                isEditingComponent && linkMode === "points"
-                  ? "visible"
-                  : "none",
+              ...mapStyles["project-lines"].layerProps.layout,
+              visibility: isViewingComponents ? "visible" : "none",
+            },
+          }}
+        />
+        <Layer
+          beforeId="street-labels"
+          {...{
+            ...mapStyles["project-lines-muted"].layerProps,
+            layout: {
+              ...mapStyles["project-lines-muted"].layerProps.layout,
+              visibility: shouldShowMutedFeatures ? "visible" : "none",
             },
           }}
         />
       </Source>
 
       <Source
-        id="project-lines"
+        id="ctn-points"
         type="geojson"
-        data={projectLines}
-        promoteId="id"
+        data={ctnPointsGeojson}
+        promoteId={SOURCES["ctn-points"]._featureIdProp}
       >
         <Layer
           beforeId="street-labels"
-          {...mapStyles["project-lines-underlay"].layerProps}
+          {...{
+            ...mapStyles["ctn-points-underlay"].layerProps,
+            layout: {
+              visibility: isEditingPoints ? "visible" : "none",
+            },
+          }}
         />
         <Layer
           beforeId="street-labels"
-          {...mapStyles[
-            clickedComponent || isEditingComponent
-              ? "project-lines-muted"
-              : "project-lines"
-          ].layerProps}
```

Comment:
I didn't catch this conditional render in the last PR so I split this into two layers (`project-lines-muted` and `project-lines`) and toggle the visibility just like other layers.